pvlib/pvsystem.py

```diff
@@ -574,7 +566,7 @@ def scale_voltage_current_power(self, data):
 
         Returns
         -------
-        scaled_data: DataFrame
+        scaled_data: DataFrame or (ordered) dict
```

Comment:
good to add ``dict``, but no need to specify ordered. I think we'll be getting rid of OrderedDict as python 3.6+ becomes the standard.

Reply:
we've used ``dict-like``  elsewhere and it applies here as well. so that's another option.